lib/tasks/geckoboard.rake

```diff
@@ -0,0 +1,7 @@
+namespace :geckoboard do
+  desc "Reset and recreate Geckoboard claims dataset"
+  task reset: :environment do
+    Claim::GeckoboardDataset.new(nil).delete
```

Comment:
Maybe a comment explaining why we're passing `nil` as the argument (eg "we don't have any data to send, we just need the object to manipulate")? It's not immediately clear what's going on here.

Reply:
Good call 👍

Reply:
I've actually tweaked the class now to make the claims an optional parameter, which makes more sense, rather than passing in `nil`